packages/manager/modules/telecom-dashboard/src/identity-check/form/confirm/confirm.controller.js

```diff
@@ -0,0 +1,14 @@
+export default class IdentityCheckFormConfirmCtrl {
```

Comment:
**question:** Can this modal be stateful? (As we tend to avoid using `$uibModalInstance` from controllers).

Reply:
IMO there is no need to add additional state code for this simple confirmation popup as it contains no business logic inside to be shared through url

![image](https://user-images.githubusercontent.com/3984480/147061610-81e2b4b2-8698-463f-b6a7-5698242d99df.png)